### PR TITLE
FBI Most Wanted

### PR DESCRIPTION
**Pull request by abel-tefera**
### Summary

- Consumed the FBI's most wanted API for displaying numeric rewards for information
- Used Redux toolkit to compose actions and reducers inside the Wanted Slice
- Created the UI by following design guidelines, using tailwind css

### CODE REVIEW

File: src/__tests__/Header.test.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { BrowserRouter } from 'react-router-dom';
+import renderer from 'react-test-renderer';
+import Header from '../components/Header';
+
+describe('tests for header component', () => {
+  test('snapshot for header', () => {
+    const component = renderer
+      .create(
+        <BrowserRouter>
```
Comment (Davidosky007) [inline src/__tests__/Header.test.js:9]: * Kindly add more tests for your components for more coverage as it's [required  ](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20unit%20tests,the%20Redux%20Store.)

File: src/index.css [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
 @tailwind components;
 @tailwind utilities;
+
+body {
+  background-color: #4369b2;
+}
+
+footer {
+  position: fixed;
+  left: 0;
```
Comment (Davidosky007) [inline src/index.css:11]: * Kindly stick to the font family given in the design 
image

File: src/redux/wanted/wantedSlice.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+            subjects: wanted.subjects,
+            title: wanted.title,
+            warning_message: wanted.warning_message,
+            weight: wanted.weight,
+            wanted_id: wanted.uid,
+            description: wanted.description,
+          })))
+          .flat()
+          .filter(({ reward_max }) => reward_max > 0);
+        // .sort((a, b) => b.reward_max - a.reward_max);
```
Comment (Davidosky007) [inline src/redux/wanted/wantedSlice.js]: * Kindly remove all commented code as it makes your codebase bulky and unprofessional

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @abel-tefera 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* All tests passed 👍 
* React router used 👍 
* Redux connected properly 👍 
* Professional looking video presentation 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
Hi @abel-tefera, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Correct Gitflow ✔️ 
- [ ] All tests passed ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**